### PR TITLE
[jquery] Copied event properties are writable.

### PR DESCRIPTION
**Pull request by leonard-thieu**
Please fill in this template.

- [x] Use a meaningful title for the pull request. Include the name of the package modified.
- [x] Test the change in your own code. (Compile and run.)
- [x] Add or edit tests to reflect the change. (Run with `npm test`.)
- [x] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [x] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [x] Run `npm run lint package-name` (or `tsc` if no `tslint.json` is present).

Select one of these and delete the others:

If changing an existing definition:
- [x] Provide a URL to documentation or source code which provides context for the suggested changes: 
http://jsfiddle.net/leonard_thieu/xpvt214o/870442/
- ~~Increase the version number in the header if appropriate.~~
- ~~If you are making substantial changes, consider adding a `tslint.json` containing `{ "extends": "dtslint/dt.json" }`.~~

---

##### Copied event properties are writable. (fbbb986a30902d69b4da0771bba091b3223e41b8)

Fixes https://stackoverflow.com/questions/49892574/trigger-a-jquery-3-event-with-ctrlkey-set.

### CODE REVIEW

File: types/jquery/index.d.ts [hunk 1]
```diff
@@ -61847,63 +61876,108 @@ jQuery.easing.easeInCubic = function ( p ) {
...
-        namespace: string;
+        pageY: number;
+        screenX?: number;
+        screenY?: number;
+        /** @deprecated */
+        toElement?: Element;
+
+        // #endregion
+
+        // region PointerEvent
```
Comment (andy-ms) [inline types/jquery/index.d.ts]: Any reason you open the region twice? Is there some way our tooling isn't working well for you here?

Comment (leonard-thieu) [reply types/jquery/index.d.ts]: WebStorm uses `// #region` to designate a region start while VS Code uses `// region`. They can both (must, in this case) be terminated using `// #endregion`. Using both `// #endregion` and `// endregion` results in odd behavior in both editors.

Comment (andy-ms) [reply types/jquery/index.d.ts]: I wonder if we could support `// #region` as well for compatibility. @uniqueiniquity

Comment (uniqueiniquity) [reply types/jquery/index.d.ts]: @andy-ms We do use `// #region`, not `// region`. Not sure why VS code would be picking up the latter and not the former.

Comment (uniqueiniquity) [reply types/jquery/index.d.ts]: See [here](https://github.com/Microsoft/TypeScript/blob/a4a5b3806e92ddced9f82b2c341e78bd91c041e4/src/services/outliningElementsCollector.ts#L86) for specifics.

Comment (andy-ms) [reply types/jquery/index.d.ts]: @leonard-thieu I can confirm that it's working in vscode with only `// #region` comments, no `// region`.

Comment (leonard-thieu) [reply types/jquery/index.d.ts]: Sorry, I had it reversed. WebStorm uses `// region`.

#### WebStorm using `// region`
image

#### WebStorm using `// #region`
image

Comment (andy-ms) [reply types/jquery/index.d.ts]: Maybe we should add support in typescript without the `#` then.

**Pull request merged**